editorconfig-eclipse-plugin/editorconfig-core-java

```diff
@@ -1 +0,0 @@
-Subproject commit 9ff6e78159540f5c305551237fcf2c7f14701b1d
```

Comment:
Oh! It must be because you have moved the dir that contains a submodule.

You will need to edit the ".gitmodules" file and update the path to the submodule. It will look like:

```
[submodule "editorconfig-core-java"]
	path = org.eclipse.editorconfig.core/editorconfig-core-java
	url = git@github.com:editorconfig/editorconfig-core-java.git
```



This is a bit of a mess but unfortunately it's required since the java editorconfig parser is not  published as a maven artifact.